src/main/java/seedu/dengue/model/predicate/FindPredicate.java

```diff
@@ -27,7 +27,7 @@ public class FindPredicate extends PredicateUtil<Person> {
      * A class that represents the predicate which is used to filter through the persons list to test for whether
      * the person belongs to the filtered list.
      * @param name (Optional) Name of the input passed in by user.
-     * @param subPostal (Optional) Substring of a Postal of the input passed in by user.
+     * @param subPostal (Optional) Substring of a {@Code Postal} of the input passed in by user.
```

Comment:
JavaDoc tags are case-sensitive, according to documentation. Might be good to drop it to lowercase.

```suggestion
     * @param subPostal (Optional) Substring of a {@code Postal} of the input passed in by user.
```